### ISSUE TITLE
Implement planemo test --number 2

### ISSUE THREAD

**Issue by frederikcoppens** [galaxyproject/planemo#262]
If you change one specific test, you want to only run that one specific tests, not all of them

**Comment by martenson** [comment]
this might be the same feature as @jmchilton meant with https://github.com/galaxyproject/planemo/issues/257
We need more description from @jmchilton before we can decide though.

**Comment by peterjc** [comment]
This (#262) is picking tests by their index, e.g. the third test.

But #257 was just testing

**Comment by jmchilton** [comment]
Yeah - I think this is actually even better than ``--new`` - given that any implementation of ``--new`` would be a little hacky.

**Comment by mvdbeek** [comment]
For this particular pattern you could also use `--failed`, but I agree that would be good.

**Comment by mvdbeek** [comment]
This would help with https://github.com/galaxyproject/galaxy-language-server/issues/159. Some of the other testing scripts (`galaxy-tool-tests` from galaxy-tool-util) and planemo itself (via the failed tests option) support this, this would probably not be a large change.

**Comment by bernt-matthias** [comment]
Might this also be useful for balancing CI chunking even more: https://github.com/galaxyproject/tools-iuc/issues/2959

**Comment by mvdbeek** [comment]
Done in https://github.com/galaxyproject/planemo/pull/1614
